2 table.sql

```diff
@@ -0,0 +1,31 @@
+use ArmyRecordsDB
+go 
+drop table if exists dbo.Soldiers
+go 
+create table dbo.Soldiers(
+    SoldiersID int not null identity primary key,
+    SoldierLastName varchar(25) not null
+        constraint c_Soldiers_soldier_last_name_cannot_be_blank check(SoldierLastName <> ''),
+    SoldierFirstName varchar(25) not null
+        constraint c_Soldiers_soldier_first_name_cannot_be_blank check(SoldierFirstName <> ''),
+    SoldierSSN char(10) not null
+        constraint c_Soldiers_soldier_SSN_cannot_be_blank check(SoldierSSN <> '')
+        constraint u_Soldiers_soldier_SSN_ unique,
+    SoldierDOB date not null
+        constraint c_Soldiers_soldier_DOB_cannot_be_blank check(SoldierDOB <> ''),
```

Comment:
Why don't you allow a soldier to be born on 01/01/1900? This is the date returned from a blank date.

Reply:
I deleted the DOB cannot be blank constraint